ert_shared/services/_base_service.py

```diff
@@ -188,6 +188,9 @@ def _read_conn_info(self, proc: Popen) -> Optional[str]:
 
             # Timeout reached, exit with a failure
             if ready == ([], [], []):
+                print(
+                    f"Service {self._service_name} startup exceeded defined timeout {self._timeout}s, initiating shutdown "
```

Comment:
Intentional space at the end of the string?

Reply:
Nope!, Fixed